features/netsocket/cellular/generic_modem_driver/OnboardCellularInterface.cpp

```diff
@@ -66,9 +66,7 @@ void OnboardCellularInterface::modem_power_down()
 #ifdef ONBOARD_CELLULAR_INTERFACE_AVAILABLE
 MBED_WEAK CellularBase *CellularBase::get_target_default_instance()
 {
-    static OnboardCellularInterface cellular;
-
-    return &cellular;
+    return mbed::CellularContext::get_default_instance();
```

Comment:
The original code was logically correct - use `OnboardCellularInterface` if the define says it exists.  That would work with someone else's `OnboardCellularInterface`. Do you need to specialise for your `CellularContext`? You should still be providing `OnboardCellularInterface` for backwards compatibility.

Reply:
changed back to original